examples/model_compress/speedup.md

```diff
@@ -0,0 +1,63 @@
+# Speedup Results
+
+*This feature is still in Alpha version.*
+
+This code only works on torch 1.3.1 and torchvision 0.4.2
+
+## slim pruner example
+
+on one V100 GPU,
+input tensor: `torch.randn(64, 3, 32, 32)`
+
+|Times| Mask Latency| Speedup Latency |
+|---|---|---|
+| 1 | 0.011968851089477539 | 0.005106925964355469 |
+| 2 | 0.020199298858642578 | 0.008769512176513672 |
```

Comment:
1. the times are small,  maybe 100, 1000, 10000, etc?
2. the accuracy is too high, 6 digits are enough.

Reply:
good suggestion, will update doc

Reply:
new experiment numbers will be added in another pr